boxsdk/util/translator.py

```diff
@@ -191,5 +193,28 @@ def translate(self, session, response_object):
 
         return translated_obj
 
+    def untranslate(self, value):
```

Comment:
This doesn't need to be a method on the `Translator` class. It doesn't actually use any of the other methods or data of the `Translator`. Additionally, we don't have a `Translator` instance available (besides the `_default_translator`) in the place where we need to use this method.

This can be either a `@classmethod` for `Translator`, or it can be a private `@classmethod` on `BaseAPIJSONObject`. I somewhat prefer the latter, because then the logic for `response_object` exists in a single place, and because it doesn't involve exposing a new public method that we'll need to maintain.